release-1.9/release-1.9.md

```diff
@@ -0,0 +1,90 @@
+# Proposed timeline for 1.9
+We will begin the 1.9 release the first week of October 2017.
+
+The proposal below is similar in layout previous releases:
+- as before, key days aren't Fridays, since it can be hard to end milestones right up against weekends
+- there are a number of North American holidays during Q4, so this release will
+  have reduced scope and fewer features
+- same as in 1.8
+  * alpha's should not be cut unless all master-release-blocking tests are passing
+  * release notes for planned features done 2 weeks out from feature freeze
+  * docs PRs deadlines ahead of release
+
+## 1.9 Release Schedule Key Dates
+- Oct 5nd (Weds) coding start
+- Oct 11th (Weds): v1.9.0-alpha.1
```

Comment:
Who uses these and why are they important.  I still don't know myself.

Reply:
@jdumars @david-mcmahon any comments on why we cut the alphas?

Reply:
a chance to see if we actually have all tests green, a chance to exercise anago before the actual release branch is cut.. if we are abandoning the idea of gating on passing tests, then I wouldn't bother cutting bits until code freeze

Reply:
Why we cut alphas?  There had been grumblings in the past on the lists when they weren't cut every 2 weeks, but if they're not used anymore, certainly the process can and should change with it.  The test signal has become less clear (red) since the gating test list expanded to include what appears to be more nice-to-haves vs. a true blocking set.

Reply:
Yes, all of that.  In some ways this process is ahead of its time.  In future releases as we get more architectural maturity and equilateral distribution of dependent projects, alpha signal will be critical.  Right now, however, I am questioning it myself other than as an internal exercise.  We never had a green alpha.  I don't even know if it's possible without a hard steer into testing as first class citizen in the release process.  Undoubtedly, that would slow velocity and probably harm the contributor base as people have to constantly context switch to fix tests.  I am fine with limiting alphas, or better yet no alphas unless we have green signal in blocking tests.

Reply:
If I recall correctly, @luxas was one of the ones who asked for alphas to be cut. I think it was because testing something downstream (kubeadm?) required released artifacts to point at.

Reply:
Yeah, but that was quite some time ago. Not required for kubeadm anymore, but I think this is pretty important for all the other downstream consumers we have.

I think this is something we should continue to do. And this time with green tests :)
We don't necessarily have to do the alphas every two weeks, but at least three alphas in two months are required IMO